specification/common/Parameters.json

```diff
@@ -0,0 +1,37 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "parameters": {
```

Comment:
These are available here too: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json#L258.

Reply:
Fixed. But since the "ResourceGroupNameParameter" defined in https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json, has  "x-ms-parameter-location": "client", which will take breaking change to storage SDK, I define it in storage.json. We can revert to use common type when the definition difference of  "ResourceGroupNameParameter" is resolved.